packages/node-cli/src/commands/object/import/ImportObjectsAction.js

```diff
@@ -59,20 +66,29 @@ module.exports = class ImportObjectsAction extends BaseAction {
 
 		try {
 			const flags = [];
+			let scriptIdArray;
 			if (this._runInInteractiveMode) {
 				if (params[ANSWERS_NAMES.IMPORT_REFERENCED_SUITESCRIPTS] !== undefined && !params[ANSWERS_NAMES.IMPORT_REFERENCED_SUITESCRIPTS]) {
 					flags.push(COMMAND_FLAGS.EXCLUDE_FILES);
 					delete params[ANSWERS_NAMES.IMPORT_REFERENCED_SUITESCRIPTS];
 				}
+
+				scriptIdArray = params[ANSWERS_NAMES.SCRIPT_ID].split(' ');
 			} else {
 				this._log.info(NodeTranslationService.getMessage(WARNINGS.OVERRIDE));
 				if (params[COMMAND_FLAGS.EXCLUDE_FILES]) {
 					flags.push(COMMAND_FLAGS.EXCLUDE_FILES);
 					delete params[COMMAND_FLAGS.EXCLUDE_FILES];
 				}
+
+				if (params[ANSWERS_NAMES.OBJECT_TYPE] === IMPORT_OBJECTS_COMMAND_TYPE_PARAM_ALL) {
+					scriptIdArray = (await this._getAllScriptIds(params)).map((el) => el.scriptId).join(' ').split(' ');
+				} else if (params[ANSWERS_NAMES.OBJECT_TYPE] !== IMPORT_OBJECTS_COMMAND_TYPE_PARAM_ALL
+					&& params[ANSWERS_NAMES.SCRIPT_ID] === IMPORT_OBJECTS_COMMAND_SCRIPT_ID_PARAM_ALL) {
```

Comment:
I think `params[ANSWERS_NAMES.OBJECT_TYPE] !== IMPORT_OBJECTS_COMMAND_TYPE_PARAM_ALL` can be removed in this statemen because it will be always `true` after the first `if`, right?

Reply:
You are correct. Updated.